site_scons/env_modules.py

```diff
@@ -63,20 +63,20 @@ def _module_func(self, command, *arguments):  # pylint: disable=no-self-use
 
         stdout, stderr = proc.communicate()
 
-        # pylint: disable=exec-used
         if sys.version_info[0] > 2:
```

Comment:
I think this was to get scons running on python2 and 3, it could probably be simplified now, but outside the scope of this PR.